src/Microsoft.AspNet.Mvc.Core/Logging/DefaultActionSelectorLoggerExtensions.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNet.Mvc.ActionConstraints;
+using Microsoft.Extensions.Logging;
+
+namespace Microsoft.AspNet.Mvc.Logging
+{
+    public static class DefaultActionSelectorLoggerExtensions
```

Comment:
Since this isn't internal and it'll be public facing, are we postponing docing this stuff till later?